### PR TITLE
Added a method that reads in data from a file

### PR DESCRIPTION
**Pull request by Bamash**

### CODE REVIEW

File: emulator.py [hunk 1]
```diff
@@ -27,9 +28,16 @@ def __init__(self):
...
+    def read_data(self, file):
+        infile = open(file,'rb')
+        new_dict = pickle.load(infile)
+        infile.close()
+        
+        for n, i in enumerate(new_dict["input_data"]):
+            self.x.append(new_dict["input_data"].values[n][1:])
+            
+        for data in new_dict["output_data"]:
+            self.x.append(new_dict["input_data"].values[n][1:])
```
Comment (Catarina-Alves) [inline emulator.py]: Shouldn't it be `self.y.append(new_dict["output_data"].values[n][1:])` in this line? (replace `x`->`y` and `input`->`output`)

**Review by Catarina-Alves — CHANGES_REQUESTED** [review]
I have a question.

File: emulator.py (hunk 1 above)
Comment (Bamash) [reply emulator.py]: Made the changes below.

**Review by Catarina-Alves — APPROVED** [review]
Seems good now.

**Comment by Bamash** [comment]
I've added some more changes to the base class. What I've essentially done is turn Kate's preprocessing and plotting code from the ANN notebook into methods for the class.

**Pull request merged**